### ISSUE TITLE
All components should have a standard focus outline

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri-design-system#257]
## Expected behavior

Follow-up from #131:

* all components should have a rectangular focus outline with sharp corners
* the outline's inner padding should be spaced away from the rest of the element so it visually surrounds it
* drop-down buttons should only have a single 'clickable area' and not receive a second 'tab' focus

## Actual behavior

While testing in the latest 0.15 branch, still seeing a fair amount of inconsistency including:

* border radius varies and in one situation is a circle
* some components are still missing outlines
* some components appear to have unnecessary secondary clickable areas

## Additional information

![2021-09-01 09 59 12](https://user-images.githubusercontent.com/2367265/131693267-438ded2f-a476-4bf6-9de2-e25b181912e7.gif)

## Environment

Mac OS, firefox, latest 0.15

Noticed on the facility user screen. A full audit might be in order.

**Comment by indirectlylit** [comment]
This only seems to be an issue on Firefox, where apparently the outline always follows the `border-radius` prop:

https://bugzilla.mozilla.org/show_bug.cgi?id=593717
